test/mocks/websocket-mock.ts

```diff
@@ -0,0 +1,25 @@
+import * as events from 'events';
+
+export class WebSocketConnection extends events.EventEmitter {
```

Comment:
```suggestion
export class WebSocketConnectionMock extends events.EventEmitter {
```

Reply:
same with the other classes in this mock file

Reply:
You may need to map the names from `WebSocketConnectionMock` to `WebSocketConnection` in order for the mocks to replace the originals.